EKF/control.cpp

```diff
@@ -1083,12 +1083,11 @@ void Ekf::rangeAidConditionsMet()
 {
 	// if the parameter for range aid is enabled we allow to switch from using the primary height source to using range finder as height source
 	// under the following conditions
-	// 1) Vehicle is in-air
-	// 2) Range data is valid
-	// 3) Vehicle is no further than max_hagl_for_range_aid away from the ground
-	// 4) Ground speed is not higher than max_vel_for_range_aid
-	// 5) Terrain estimate is stable (needs better checks)
-	if (_control_status.flags.in_air && !_rng_hgt_faulty) {
+	// 1) Range data is valid
```

Comment:
This change is not required to fix the 'invisible ceiling; defect. It increase vulnerability to range finder obscuration and tilt when the vehicle is being handled prior to takeoff.